spec/models/users/setting_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rails_helper"
+
+RSpec.describe Users::Setting, type: :model do
+  describe "validations" do
+    subject { create(:setting) }
+
+    let(:user) { create(:user) }
+    let(:setting) { create(:setting, user: user) }
+
+    it { is_expected.to validate_inclusion_of(:inbox_type).in_array(%w[open_inbox private_inbox]) }
```

Comment:
If we remove the validations in the model as I outlined above, we can also remove this line and replace it with some validation for the enums:
Example:

```suggestion
    it { is_expected.to define_enum_for(:inbox_type).with_values(private: 0, open: 1).with_suffix(:inbox) }
    it { is_expected.to define_enum_for(:config_font).with_values(comic_sans: 1, monospace: 2, open_dyslexic: 3, sans_serif: 4, serif: 5) }
```
and you can add others for the other enums